tools/pontoon/users.md

```diff
@@ -0,0 +1,65 @@
+# Managing users and profile settings
```

Comment:
I agree on using "user settings" (it's clearer than "profile"), but this section is exactly about that page, so I'm not sure there's confusion.